addons/stock_account/tests/test_stockvaluationlayer.py

```diff
@@ -335,6 +336,27 @@ def test_currency_precision_and_standard_svl_value(self):
         finally:
             self.env.user.company_id = old_company
 
+    def test_change_qty_and_locations_of_done_sml(self):
+        sub_stock_loc = self.stock_location.child_ids[0]
```

Comment:
@adwid @clesgow 

Hello :)

This line crashes without demo data https://runbot.odoo.com/runbot/build/71925344
Could you please fix it in 17.4 and in the forward-ports?

I'm assigning you to the runbot error

Thanks :)

Reply:
Woooooooooops :see_no_evil: 
Sorry!
https://github.com/odoo/odoo/pull/190689